pyiode/iode_database/variables_database.pyx

```diff
@@ -948,24 +949,46 @@ cdef class Variables(_AbstractDatabase):
             return [cpp_period.decode() for cpp_period in 
                     self.database_ptr.get_list_periods(from_period.encode(), to_period.encode())]
         
-    def low_to_high(self, type_of_series: str, method: str, filepath: str, var_list: Union[str, List[str]]):
+    def low_to_high(self, type_of_series: str, method: str, filepath: Union[str, Path], var_list: Union[str, List[str]]):
         """
-        Transform low periodicity series to high periodicity series (i.e. variables). 
+        Build series with higher periodicity for *stock data* (Unemployment, Debt, ...) or 
+        *flow data* (GNP, Deficit, ...).
+
+        The list of specified series (variables) from the input file are loaded into the current 
+        Variables database and the periodicity of these series (variables) is modified simultaneously. 
+        The new periodicity is the one currently defined in the current Variables database.
+
+        The loaded series are added to or replace those (for existing names) in the current 
+        Variables database.
+
+        This procedure exists for the following cases:
+
+            - annual to monthly 
+            - annual to quarterly 
+            - quarterly to monthly 
+        
+        Two types of series are available, one for stocks ('L'), the other for flows ('S').
+
+        Three interpolation methods are available:
+
+            - linear ('L'): A[1980Q{1,2,3,4}] = A[1979Y1] + i * (A[1980Y1] - A[1979Y1])/4 i = 1,2,3,4 
+            - cubic splines ('C'): cubic interpolation 
+            - step ('S') : A[1980Q{1,2,3,4}] = A[1980Y1] 
 
         Parameters
         ----------
         type_of_series : str
             Two types of series are considered: 'stock' and 'flow':
-                - 'S' (LTOH_STOCK) : the interpolated values are of the same order of magnitude as the original values
-                - 'F' (LTOH_FLOW) : the values of the sub-periods are additive over a period
+                - 'S' (LTOH_STOCK): stock data (Unemployment, Debt, ...)
+                - 'F' (LTOH_FLOW): flow data (GNP, Deficit, ...)
```

Comment:
"the values of the sub-periods are additive over a period"

This sentence was useful IMO.